presto-docs/src/main/sphinx/develop/password-authenticator.rst

```diff
@@ -37,9 +37,12 @@ Example configuration file:
 
 .. code-block:: none
 
-    password-authenticator.name=custom-access-control
+    password-authenticator.name=custom-password-authenticator
     custom-property1=custom-value1
     custom-property2=custom-value2
 
 Additionally, the coordinator must be configured to use password authentication
 and have HTTPS enabled.
+
+Add the property shown below to the coordinator's ``config.properties`` file:
```

Comment:
```suggestion
Add the property shown below to the coordinator's ``config.properties`` file::

```

Reply:
Thanks for the feedback! I'll format the code example in the last line and share an updated version.